fhir-term-graph/src/main/java/com/ibm/fhir/term/graph/loader/impl/UMLSTermGraphLoader.java

```diff
@@ -47,46 +49,40 @@
     private static final String UMLS_CONCEPT_NAMES_AND_SOURCES_FILE = "MRCONSO.RRF";
     private static final String UMLS_SOURCE_INFORMATION_FILE = "MRSAB.RRF";
     private static final String UMLS_RELATED_CONCEPTS_FILE = "MRREL.RRF";
-
     private static final String UMLS_DELIMITER = "\\|";
+    public static final String COS_BUCKET_NAME = "COS_BUCKET_NAME";
 
-    // Map to track AUI to SCUI relationships, since MRREL uses AUI, but granularity of concepts used in MRCONSO is at SCUI level
-    private Map<String, String> auiToScuiMap = new ConcurrentHashMap<>(1000000);
+    // Map to track AUI to SCUI relationships, since MRREL uses AUI, but granularity of concepts used in MRCONSO is at SCUI/CODE level
+    private Map<String, String> auiToCodeMap = new ConcurrentHashMap<>(1000000);
```

Comment:
For consistency, should this use caffeine? or is it fine to use concurrenthashmap as it sits outside of the server. 

Also, it may be worthwhile tweaking the shell script to set a decently high jvm memory args -Xmx 4G (I only saw 512m on elastic search)

Reply:
This map unfortunately needs to keep all entries in memory for later reference, so a cache isn't really applicable.

I have updated the load script to default to 4G max memory.

Reply:
Caffeine is in memory, it's highly efficient, and manages the size appropriately and expiry

Reply:
My understanding is Caffeine is aimed at caching (with expiration). Since there's no room for expiration here, I don't think it's necessary.  Is there some benefit I'm not thinking of?